packages/frontend/core/src/modules/db/entities/table.ts

```
@@ -19,10 +19,23 @@ export class WorkspaceDBTable<
     super();
   }
 
+  isReady$ = LiveData.from(
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)